### PR TITLE
Packaging specfile clean up

### PR DESCRIPTION
**Pull request by maxamillion**
- Remove no longer needed defattr
    - Remove negative comments
    - Add %license where needed
    - Fix dir ownership for sdn
    - Add better description

### CODE REVIEW

File: origin.spec [hunk 1]
```diff
@@ -65,7 +65,11 @@ Obsoletes:      openshift < %{package_refector_version}
 ### AUTO-BUNDLED-GEN-ENTRY-POINT
 
 %description
-%{summary}
+Origin is a distribution of Kubernetes optimized for enterprise application
+development and deployment, used by OpenShift 3 and Atomic Enterprise. Origin
+adds developer and operational centric tools on top of Kubernetes to enable
+rapid application development, easy deployment and scaling, and long-term
+lifecycle maintenance for small and large teams and applications.
```
Comment (sdodson) [inline origin.spec]: Just curious, is this original or copied from somewhere?

Comment (maxamillion) [inline origin.spec]: Yes, first paragraph of https://github.com/openshift/origin/blob/master/README.md minus the first sentence.

**Comment by maxamillion** [comment]
@tdawson @sdodson @eparis Please review. :)

File: origin.spec [hunk 2]
```diff
@@ -303,6 +309,8 @@ install -p -m 644 contrib/completions/bash/* %{buildroot}%{_sysconfdir}/bash_com
 %{_sysconfdir}/bash_completion.d/oadm
 %{_sysconfdir}/bash_completion.d/openshift
 %dir %config(noreplace) %{_sysconfdir}/origin
+%ghost %dir %config(noreplace) %{_sysconfdir}/origin
```
Comment (sdodson) [inline origin.spec]: Isn't this a dupe of line 311?

File: origin.spec [hunk 3]
```diff
@@ -367,35 +375,36 @@ fi
...
-%systemd_preun %{basename:%{name}-node.service}
+%systemd_preun %{name}-node.service
 
 %postun node
 %systemd_postun
 
 %files sdn-ovs
-%defattr(-,root,root,-)
+%dir %{_unitdir}/docker.service.d/
+%dir %{_unitdir}/%{name}-node.service.d/
```
Comment (sdodson) [inline origin.spec]: If we own both of these and someone else places files in them do we delete them on uninstall?

File: origin.spec [hunk 4]
```diff
@@ -417,27 +426,28 @@ if [ "$1" = 0 ]; then
...
+%license LICENSE
 %{_bindir}/oc
 %{_bindir}/kubectl
 %{_sysconfdir}/bash_completion.d/oc
 
 %if 0%{?make_redistributable}
 %files clients-redistributable
+%dir %{_datadir}/%{name}/linux/
+%dir %{_datadir}/%{name}/macosx/
+%dir %{_datadir}/%{name}/windows/
```
Comment (sdodson) [inline origin.spec]: I had done this in https://github.com/openshift/origin/pull/6257 as well, I'll pull it out of my PR as it's somewhat tangential to the issue I'm addressing there.

**Comment by tdawson** [comment]
Let's merge this one first, then work on getting #6257 merged.

**Comment by tdawson** [comment]
[merge]

**Comment by tdawson** [comment]
The tests are going to continually fail due to a conflict in the centos test between systemd and systemd-container.
I am going to manually merge.

**Pull request merged**